projects/components/src/popover/popover-ref.ts

```diff
@@ -25,9 +25,9 @@ export class PopoverRef {
   private _visible: boolean = false;
 
   public constructor(
-    private readonly overlayRef: OverlayRef,
-    private readonly positionBuilder: PopoverPositionBuilder,
-    private readonly navigationService: NavigationService
+    public readonly overlayRef: OverlayRef,
+    public readonly positionBuilder: PopoverPositionBuilder,
```

Comment:
Why do these need to be exposed?

Reply:
this was done for the constructor in ModalRef. If we are moving to an independent ModalRef, this won't be required.